src/components/Modal/Modal.module.scss

```diff
@@ -5,7 +5,7 @@
 }
 
 .container {
-  z-index: 2;
+  z-index: 10000;
```

Comment:
@eran-cohen why not have a css variable that user can override ?

Reply:
@orrgottlieb we will, I'll add a task for it, just want to release this asap because it's blocking someone from using the modal

Reply:
sure